# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/libelf/hotfix.patch.musl
# Copyright (C) 2023 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- elfutils-0.190/libelf/gelf.h.vanilla	2023-12-16 11:59:43.322298065 +0100
+++ elfutils-0.190/libelf/gelf.h	2023-12-16 12:00:39.854295651 +0100
@@ -83,7 +83,9 @@
 typedef Elf64_Rela GElf_Rela;
 
 /* Relative relocation entry (in section of type SHT_RELR).  */
+#if 0
 typedef Elf64_Relr GElf_Relr;
+#endif
 
 /* Program segment header.  */
 typedef Elf64_Phdr GElf_Phdr;
